tests/other_test.go

```diff
@@ -22,3 +24,44 @@ func TestRunRestoredDBExample(t *testing.T) {
 	assert.Nil(t, err, "This should not have errored")
 	assert.NotNil(t, output, "Expected some output")
 }
+
+func TestRunAutoscaleExample(t *testing.T) {
+	t.Parallel()
+
+	options := testhelper.TestOptionsDefaultWithVars(&testhelper.TestOptions{
+		Testing:            t,
+		TerraformDir:       autoscaleExampleTerraformDir,
+		Prefix:             "pg-autoscale",
+		BestRegionYAMLPath: regionSelectionPath,
+	})
+
+	output, err := options.RunTestConsistency()
+	assert.Nil(t, err, "This should not have errored")
+	assert.NotNil(t, output, "Expected some output")
+}
+
+func testRunReplicaExample(t *testing.T, version string) {
+	t.Parallel()
+
+	options := testhelper.TestOptionsDefaultWithVars(&testhelper.TestOptions{
+		Testing:            t,
+		TerraformDir:       replicaExampleTerraformDir,
+		Prefix:             "pg-replica",
+		BestRegionYAMLPath: regionSelectionPath,
+		TerraformVars: map[string]interface{}{
+			"pg_version": version,
+		},
+	})
+
+	output, err := options.RunTestConsistency()
+	assert.Nil(t, err, "This should not have errored")
+	assert.NotNil(t, output, "Expected some output")
+}
+
+func TestRunReplicaExample(t *testing.T) {
+	t.Parallel()
+	versions := []string{"11", "12", "13", "14"}
```

Comment:
No need to run against all versions here. Thats far too many tests. We already test all versions in pr_test.go in `TestRunCompleteExample`.  Just don't pass any value for `pg_version` variable so the default version gets used

Reply:
Okay. Made the changes as suggested.